Makefile

```diff
@@ -403,8 +404,14 @@ env.d/development:
 env.d/lambda:
 	cp env.d/lambda.dist env.d/lambda
 
-help:  ## Show this help
+h: # short default help task
```

Comment:
```suggestion
h: ## Show short help
```
Just if you didn't mean to "hide" this task

Reply:
Yes that was intentional to not display it.

Reply:
:+1: